packages/account/src/Components/additional-kyc-info-modal/additional-kyc-info-modal.scss

```diff
@@ -0,0 +1,101 @@
+.dc-modal {
+    &-header {
+        border-bottom: 2px solid var(--general-section-1);
+    }
+
+    &-footer {
+        padding: 1.6rem 0 0;
+        margin: unset;
+    }
+}
+
+.dc-page-overlay {
+    &__header {
+        padding: 2.5rem 2rem;
+
+        &-close {
+            position: inherit;
+            right: 0;
+        }
+    }
+}
+
+.additional-kyc-info-modal {
+    &__portal {
+        &-header {
+            font-size: 1.4rem;
+            text-align: left;
```

Comment:
Header is now Localize component. we can make it Text component and pass these styles as props